### DISCUSSION TITLE
Language bar missing

### DISCUSSION THREAD

**Discussion by BlackRule** [valinet/ExplorerPatcher#98] [Q&A] [answered]
I don't know why nobody tells about this and what's a solution (Quicj search didn't show me any 3rd party app/fix now).  
![image](https://user-images.githubusercontent.com/3146449/137474080-106f81ae-e9a9-45d2-964b-f9bf9b3f760e.png)

Also language switch popup is too fast (On W10 before upgrade I had registry tweak to set taskbar animations to 0 though.. smth. like this. That might be the cause)
https://user-images.githubusercontent.com/3146449/137474719-98220ff6-b82f-4992-90ec-83427ac156b5.mp4

**Comment by BlackRule** [comment]
Also the clock is TClock I don't know if this info is useful...

**Comment by valinet** [comment, marked as answer]
The language bar is not available anymore for docking in the taskbar. There is already a language selector. Why not use that?

**Reply by BlackRule** [reply]
What do you mean by "a language selector." ? Where it's supposed to be located? On my screenshot you see the Win10 lang bar is missing for unknown reason. 
I just want to see my current language inside taskbar...

**Reply by valinet** [reply]
Run this: `%windir%\explorer.exe shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9}\SystemIcons`, choose "Input indicator", make sure it is "On", it will display the language selector in the taskbar.

**Reply by BlackRule** [reply]
Thanks I was looking there carefully first but there were no such option Now it appeared (Along with Action Center)
